docs/user-guide/configure-uss.md

```diff
@@ -50,29 +57,74 @@ Users who install Zowe to run Zowe scripts need to have an OMVS segment. If the
 ## Address space region size
 
 Java as a prerequisite for Zowe requires a suitable z/OS region size to operate successfully while you install and configure Zowe. It is suggested that you do not restrict the region size, but allow Java to use what is necessary. Restricting the region size might cause failures with storage-related error messages such as the following one:
+
 ```
 JVMJ9VM015W Initialization error for library j9gc29(2)
 Error: Could not create the Java Virtual Machine.
 Error: A fatal exception has occurred. Program will exit
 ```
+
 You can fix the storage-related issue by making one of the following changes:
 
 - ASSIZEMAX parameter
 
-   The ASSIZEMAX parameter is the maximum size of the process's virtual memory (address space) in bytes. 
-   
+   The ASSIZEMAX parameter is the maximum size of the process's virtual memory (address space) in bytes.
+
    To specify the JVM maximum address space size on a per-user basis, set the ASSIZEMAX configuration parameter to the value of `2147483647`.
 
    **Note:** Running a shell script via TSO OMVS will run the shell in the TSO address space, unless you specify `_BPX_SHAREAS=NO` when invoking OMVS. If you are using TSO OMVS to install Zowe, you will need `export _BPX_SHAREAS=NO` to make the ASSIZEMAX change effective.
 
 - SIZE parameter of TSO segment
 
    Set SIZE operand of TSO segment to the value of `2096128`.
-    
+
    **Note:** If you set `export _BPX_SHAREAS=YES` in your shell setup as recommended, Java will run in the TSO address space and the SIZE change will work.
 
-- `ulimit -A` 
+- `ulimit -A`
 
    The maximum address space size for the process should be at least 250 M, in units of 1024 bytes. For example, `ulimit -A 250000`.
-    
+
    **Note:** Running `ulimit -a` displays the current process limits.
+
+## Temporary files management
+
+Zowe server components require the use of temporary files. By default, these are written to the global `/tmp` directory in the USS file system.
+This article describes options to customise the destination directory for all Zowe server components.
+
+### How to customize
+
+There are three environment variables that control the directory used to place these temporary files:
+
+- `TMPDIR`: This is the main environment variable, it controls the directory used for most USS operations.
+- `TEMP_DIR`: This variable controls...
```

Comment:
I assume you  are waiting for information which will be added here.

Reply:
Thank you, I forgot to update it